website/content/docs/release-notes/consul-api-gateway/v0_1_x.mdx

```diff
@@ -0,0 +1,72 @@
+---
+layout: docs
+page_title: 0.1.x
+description: >-
+  Consul API Gateway release notes for version 0.1.x
+---
+
+
+# Consul API Gateway 0.1.0
+
+## OVerview
+
+This is the first General Availability (GA) release of Consul API Gateway. It
+provides controlled access for network traffic, from outside a Consul service
+mesh, to Services on the mesh
+
+In Kubernetes deployments, it can be used as an alternative to Consul's Ingress
+Gateway and provides capabilities Ingress Gateway does not. These include:
+
+1. It allows you to configure and deploy new gateways at any time, without
+   rerunning the Consul Helm chart. The configuration of a running gateway can
+   be changed dynamically at anytime, usually without disrupting any the the
+   traffic flowing through it.
+1. Listeners on a gateway can use TLS server certificates signed by any
+   Certificate Authority (CA). This allows you to use certificates from public
+   CA's, like Verisign and Let's Encrypt, and private CA's, such as a company's
+   internal CA.
+
+## Release Highlights
+
+1. **Supported Runtimes:** Kubernetes - self managed, AWS EKS, Google GKE and Azure AKS.
+1. Install via the  HashiCorp Consul Helm chart.
+1. Works with self-managed Consul servers and HCP Consul servers
+1. Configure via Kubernetes Gateway API - v1alpha2
+1. Deploy 1 or more logical API Gateways per Kubernetes cluster
+1. Supported protocols: HTTP, HTTPS, TCP, TCP+TLS
+1. Supported HTTP versions: 1.1 & 2
+1. Load balance across a service’s instances
+1. Listeners load TLS certificates, signed by any CA, from Kubernetes Secret Storage
+1. Route HTTP/S traffic to Services based on matching:
+   - Hostname
+   - URI Path
+   - HTTP Header
+   - HTTP Method
+   - HTTP Query parameters
```

Comment:
I think they are proper nouns in this case because they are being used like the names of fields in a database. Technically, they are fields in the structure of a HTTP message.